extensions/wikia/Search/classes/Services/FeedEntitySearchService.php

```diff
@@ -68,31 +68,32 @@ private function createQuery( $query ) {
 			$query .= ' +host:(' . implode( ' | ', $this->hosts ) . ') ';
 		}
 
-		$query .=
-			( isset( $q ) ? ' AND +(article_quality_i:[' . $q . ' TO *])' : '' )
-			. ( isset( $l ) ? ' AND +(lang:' . $l . ')' : '' )
-			. ( isset( $wids ) ? ' AND +wid:( ' . implode( ' | ', $wids ) . ')' : '' )
-			. ( isset( $hubs ) ? ' AND +hub:( ' . implode( ' | ', $hubs ) . ')' : '' );
+		$query .= ( isset( $q ) ? ' AND +(article_quality_i:[' . $q . ' TO *])' : '' ) .
+			( isset( $l ) ? ' AND +(lang:' . $l . ')' : '' ) .
+			( isset( $wids ) ? ' AND +wid:( ' . implode( ' | ', $wids ) . ')' : '' ) .
+			( isset( $hubs ) ? ' AND +hub:( ' . implode( ' | ', $hubs ) . ')' : '' );
+
 		return $query;
 	}
 
 	protected function consumeResponse( $response ) {
-		$items = [ ];
+		$items = [];
 		foreach ( $response as $res ) {
-			$items[ ] = [
-				'id' => $res[ 'id' ],
-				'pageid' => $res[ 'pageid' ],
-				'page_id' => $res[ 'pageid' ],
-				'url' => $res[ 'url' ],
-				'title' => $res[ 'title_en' ],
-				'timestamp' => strtotime( $res[ 'created' ] ),
-				'host' => $res[ 'host' ],
-				'wid' => $res[ 'wid' ],
-				'wikia_id' => $res[ 'wid' ],
-				'wikititle' => $res[ 'wikititle_en' ],
-				'ns' => $res[ 'ns' ]
+			$items[] = [
+				'id' => $res['id'],
+				'pageid' => $res['pageid'],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "pageid" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)